tests/scripts/run-test-suites.pl

```diff
@@ -82,16 +83,18 @@ sub pad_print_center {
     print $padchar x( $padlen ), " $string ", $padchar x( $padlen ), "\n";
 }
 
-for my $suite (@suites)
+for my $suite_path (@suites)
 {
+    my $suite = $suite_path;
+    $suite =~ s!.*/!!;
     print "$suite ", "." x ( 72 - length($suite) - 2 - 4 ), " ";
     if( $suite =~ /$skip_re/o ) {
         print "SKIP\n";
         ++$suites_skipped;
         next;
     }
 
-    my $command = "$prefix$suite";
+    my $command = "$prefix$suite_path";
```

Comment:
That only works if the auto-chdir behavior in `host_test.function` is enabled. Also it doesn't work on Windows with cmd. Patch incoming.